includes/REST_API/Stories_Autosaves_Controller.php

```diff
@@ -76,11 +76,9 @@ class Stories_Autosaves_Controller extends WP_REST_Autosaves_Controller implemen
 	public function __construct( Story_Post_Type $story_post_type ) {
 		parent::__construct( $story_post_type->get_slug() );
 
-		$parent_controller = $story_post_type->get_parent_controller();
-
-		$this->parent_base       = $story_post_type->get_slug();
-		$this->parent_controller = $parent_controller;
-		$this->rest_namespace    = $story_post_type->get_rest_namespace();
```

Comment:
So what's with the `$rest_namespace` property on line 67? Don't we need that anymore?

Reply:
Good spot.

Reply:
Good catch. removd.